components/tensorflow-notebook-image/Dockerfile

```diff
@@ -86,7 +86,6 @@ ENV LANGUAGE en_US.UTF-8
 # but allow for non-initial launches of the notebook to have
 # $HOME provided by the contents of a PV
 RUN useradd -M -s /bin/bash -N -u $NB_UID $NB_USER && \
-    chown -R ${NB_USER}:users /usr/local/bin && \
```

Comment:
Why did you remove that line?
Don't we just want to add

```
chown -R ${NB_USER}:users ${HOME}
```

And then you could verify by  building the docker image and doing ls -la /home
hopefully /home/jovyan should be owned by user jovyan and group users.

Reply:
Sorry, I was really stupid. Fixed!